guides/troubleshooting/inotify-watch-limits.md

```diff
@@ -1,57 +1,54 @@
 ---
-title: Troubleshooting inotify Watcher Limit Problems
+title: inotify Watcher Limit Problems
 description: Learn how to resolve problems related to the inotify Watcher Limit.
 ---
 
-The [`inotify` facility] allows programs to monitor files for changes, so that
-they will receive an event immediately whenever a user or program modifies the
-file. Many such programs can fall back to periodically checking files for
-changes, also known as polling. Using polling avoids the watch limit, but may
-result in higher system loading and a longer interval for the program to detect
-changes.
-
-`inotify` requires kernel resources (memory and processor) for each file it
-tracks. As a result, the Linux kernel limits the number of file watchers that
-each user can register. The default settings vary according to the host system
-distribution; on Ubuntu 20.04 LTS, the default limit is 8,192 watches per
-instance.
-
-[`inotify` facility]: https://en.wikipedia.org/wiki/Inotify
-
-With some applications and tools, including Webpack or [code-server], you may
-encounter an error similar to the following:
+When using some applications and tools, including Webpack or [code-server], you
+may encounter an error similar to the following:
 
 > Watchpack Error (watcher): Error: ENOSPC: System limit for number of file
 > watchers reached, watch '/some/path'
 
 [code-server]: https://github.com/cdr/code-server
 
+This article will show you how to diagnose and troubleshoot this error, which
+relates to an elevated number of inotify watchers in use.
+
+## Background
+
+[`Inotify`] allows programs to monitor files for changes, so that they receive
+an event whenever a user or program modifies a file. `inotify` requires kernel
+resources (memory and processor) for each file it tracks. As a result, the Linux
+kernel limits the number of file watchers that each user can register. The
+default settings vary according to the host system distribution; on Ubuntu 20.04
+LTS, the default limit is 8,192 watches per instance.
+
+[`Inotify`]: https://en.wikipedia.org/wiki/Inotify
```

Comment:
Wikipedia always capitalizes the first letter of articles, but that's due to a limitation of the MediaWiki platform - the article itself shows it as "inotify"